UnitySDK/Assets/ML-Agents/Scripts/HeuristicBrain.cs

```diff
@@ -24,31 +22,25 @@ public class HeuristicBrain : Brain
         [HideInInspector]
         public MonoScript decisionScript;
 #endif
+        [FormerlySerializedAs("c_decision")]
```

Comment:
OK to rename this?

Reply:
yeah, the `FormerlySerializedAs` annotation should prevent any issues for people that upgrade to this version.